### ISSUE TITLE
Digital timestamps are given in 12h format

### ISSUE THREAD

**Issue by SebiderSushi** [virtualgeoff/sunclock#13]
Changing the `Accept-Language` header my browser is sending did not affect this either.

**Comment by virtualgeoff** [comment]
Yes. The info-text times use the JS `Date.toLocaleTimeString()` method, so they _should_ respect the users' settings, but they don't. It's browser dependent: for me, Safari and Firefox use 12 hour format, while Chrome and Edge use 24 hour, regardless of my "Language and Region" settings (MacOS).

I think I need to do the formatting myself. The clock face will always be 24 hour, but I'm not fussed about the text, so I'll make that an option, with 24 hours by default.

**Comment by royappa** [comment]
Hi Geoff, I made this change for myself here: 
https://github.com/royappa/sunclock2022/commit/f7ed8c61c60434966b797f1a72575f3c3d3740c7 

Not confident enough to submit a pull request but if any of it is helpful for your own changes, that would make me happy :-)
I host my version here: https://sunclock2022.web.app/
Love your program, thanks!

**Comment by SebiderSushi** [comment]
I revisited the issue and had success specifying the `hour12` boolean in the options parameter for `toLocaleTimeString()` in `main.js -> formatTime()` and it does exactly what i want in Firefox 108 on desktop and in Chromium 108 on Android.
```JS
return (new Date(Math.round(time/60000)*60000)).toLocaleTimeString([], { hour12:false, hour:'numeric', minute:'2-digit' });
```
Edit 2023-02-26: I just noticed that with this code the time for solar midnight is displayed as "24:xx" in Chromiun on android while it's "00:xx" on Firefox on desktop. Doesn't bother me so i didn't check why exactly that is but i guess this might be something to look out for if it's undesired behavior.
